cmake/Modules/SWIG_CGAL_Macros.cmake

```diff
@@ -1,10 +1,12 @@
+include (TargetLinkLibrariesWithDynamicLookup)
```

Comment:
Could you explain why is this needed now?

Reply:
I removed it, this is unrelated